examples/SkyTEM1D/Menindee/Read_hdr.jl

```diff
@@ -0,0 +1,23 @@
+#Steps to read and modify the SKyTEM headerfile 
+using DataFrames
+using CSV
+
+#function to read the *.dfn file and its column number and write it as a *.hdr file 
+
+function dfn2hdr(dfnfile::String,hdrfile::String)
```

Comment:
`dfn2hdr` is now in the core `transD_GP` package so use that version rather than defining in the script